src/components/Sections/AccordionSection/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, Prop } from "vue-property-decorator";
+import BaseComponent from "@/components/BaseComponent";
+import Accordion from "./Accordion";
+import { AccordionSectionProps } from "@/types/sections";
+import "./style.css";
+
+@Component({
+  name: "AccordionSection",
+})
+class AccordionSection extends BaseComponent<AccordionSectionProps> {
+  @Prop({ default: false }) dark: AccordionSectionProps["dark"];
+  @Prop({ required: true }) items!: AccordionSectionProps["items"];
+  @Prop() title: AccordionSectionProps["title"];
+
+  state = this.items.map((item, index) => {
+    return {
+      index,
```

Comment:
Please remove the index from the object since it is not used anywhere.

Reply:
You could change your component state to just represent the selected index. So you do not need to loop over the array to set the open flag of the corresponding item.

`selectedIndex: number | null = null`